src/exercises/cheri-tags/README.md

```diff
@@ -0,0 +1,46 @@
+# Demonstrate CHERI Tag Protection
```

Comment:
Here again I think I'd express this as "Demonstrate CHERI pointer integrity protection" or similar?

Or maybe it's provenance .. we should decide.